data_visualization/visualization_helper.py

```diff
@@ -0,0 +1,94 @@
+import matplotlib.pyplot as plt
+from collections import defaultdict
+from nltk.stem import PorterStemmer as stemmer
+
+def pos_counter(pos_lsts, pos_dict):
+    for pos in pos_lsts:
+        if pos[0] in pos_dict.keys():
```

Comment:
Just `pos[0] in pos_dic` will do the job for you.

Reply:
We did it! Thank you for your point.